daemon/oci_linux.go

```diff
@@ -723,7 +738,7 @@ func (daemon *Daemon) createSpec(c *container.Container) (retSpec *specs.Spec, e
 	s.Linux.CgroupsPath = cgroupsPath
 
 	if err := setResources(&s, c.HostConfig.Resources); err != nil {
-		return nil, fmt.Errorf("linux runtime spec resources: %v", err)
+		return nil, errdefs.InvalidParameter(fmt.Errorf("linux runtime spec resources: %v", err))
```

Comment:
Can you change this to `errors.Wrap`?

Reply:
Yeah, I realize I discovered that `errors.Wrap` works with errdefs, but I forgot to go back and change the ones I'd already written.

Reply:
To be clear, do you think this:

```go
errdefs.InvalidParameter(errors.Wrap(err, "linux runtime spec resources:")
```
or this:

```go
errors.Wrap(errdefs.InvalidParameter(err), "linux runtime spec resources:")
```

Reply:
I think the first one would be clearer (and the `InvalidParameter` would not be nested deeper)